.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
For best coding practices, it is better to add the `dist` folder to `.gitignore` because you can get it by running:
`npm run build`